redbot/setup.py

```diff
@@ -77,7 +78,7 @@ def get_data_dir(*, instance_name: str, data_path: Optional[Path], interactive:
             "We were unable to check your chosen directory."
             " Provided path may contain an invalid character."
         )
-        sys.exit(1)
+        sys.exit(ExitCodes.CRITICAL)
```

Comment:
This should probably be `CONFIGURATION_ERROR`, since restarting won't make the path valid.

Reply:
I think this should be `INVALID_CLI_USAGE` as it concerns current user input rather than the existing configuration which is what all `CONFIGURATION_ERROR`s are for.

`CRITICAL` doesn't seem to make much sense here though, yeah.